src/VueTyping.js

```diff
@@ -1,58 +1,102 @@
 export default {
-    render (h) {
-        return h('span', this.speechFrameFragment)
-    },
-    props: {
-        framerate: {
-            type: Number,
-            default: 1
+  render(h) {
+    return h("div", [
+      h("span", this.speechFrameFragment),
+      h(
+        "span",
+        {
```

Comment:
It should be a good idea to add a class called "typing-cursor" to the span, so the user can customize it using css too

Reply:
That's a great idea ! I will implement it.